catalog/spatial/wfs/spatial-wfs-featuretransformer/src/test/logback-test.xml

```diff
@@ -0,0 +1,24 @@
+<!--
```

Comment:
This is to avoid a Windows surefire issue where too many logs are being printed.